interfaces/builtin/raw_input.go

```diff
@@ -0,0 +1,86 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/udev"
+)
+
+const rawInputSummary = `allows access to raw input devices`
+
+const rawInputBaseDeclarationSlots = `
+  raw-input:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const rawInputConnectedPlugSecComp = `
+# Description: Allow handling input devices.
+# for udev
+bind
```

Comment:
raw_usb doesn't have bind but we are not very consistent about this apparently

Reply:
This is needed, or libinput failes to initialize udev.

Reply:
for consistency we should make a separate PR that adds this also to raw-usb